### ISSUE TITLE
Can we issue a new release of the release of this library, please?

### ISSUE THREAD

**Issue by valydia** [playframework/netty-reactive-streams#25]
In order to upgrade playframework to Netty 4.1, we need a compatible version of this library supporting Netty 4.1 as well [#6809](https://github.com/playframework/playframework/issues/6809).

**Comment by wsargent** [comment]
This is also dependent on shading the play-ws AHC dependency, which uses Netty 4.0.

**Comment by legendpeng** [comment]
+1

**Comment by wsargent** [comment]
Released 2.0.0-M1 https://oss.sonatype.org/content/repositories/releases/com/typesafe/netty/netty-reactive-streams/2.0.0-M1/

```
mvn release:prepare
mvn release:perform
```

Note that you'll need ~/.m2/settings.xml configured for the mvn-gpg-plugin and you'll also need to push your gpg key to pgp.mit.edu specifically if it's not there, i.e.

```
gpg --send-key 0xa7f14872267a1666 --keyserver pgp.mit.edu
```
